option/internaloption/internaloption.go

```diff
@@ -209,6 +209,27 @@ func (w enableNewAuthLibrary) Apply(o *internal.DialSettings) {
 	o.EnableNewAuthLibrary = bool(w)
 }
 
+// EnableAsyncRefreshDryRun  returns a ClientOption that specifies if libraries in this
+// module should asynchronously refresh auth token in parallel to sync refresh
+//
+// errHandler function will be called when there is an error while refreshing
+// the token asynchronously
```

Comment:
What should the errHandler function do? Log the error?

Reply:
No specific requirement on that. User can do whatever they need to

Reply:
Done